custom/penn_state/__init__.py

```diff
@@ -1,7 +0,0 @@
-from . import reports
-
-CUSTOM_REPORTS = (
-    ('Custom Reports', (
-        reports.LegacyReportView,
-    )), 
-)
```

Comment:
This must be from before I used an editor that added newlines to the ends of files.  Ah, memories!